content/blog/2024-05-21-Audit.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Security Audit for Eclipse Kuksa released"
+date = "2024-05-21"
+tags = ["KUKSA.val", "Audit", "Security"]
+categories = ["Implementation"]
+banner = "img/banners/2024-05-21-auditBanner.png"
++++
+
+Applications that utilize artifacts from the Kuksa project need to rely on the secure execution of these components. We are, therefore, happy to announce the publication of an external audit report carried out by [Quarkslab](https://www.quarkslab.com). The audit was facilitated through [OSTIF](https://ostif.org) and the [Eclipse Foundation](https://www.eclipse.org) and made possible by the founding the Eclipse Foundation received from the [Alpha-Omega project](https://openssf.org/community/alpha-omega/).
```

Comment:
... of Kuksa's components

Are we not more happy about having successfully performed the audit at all? Maybe: ... happy to announce that we carried out an an external audit with [Quarkslab](https://www.quarkslab.com), which is now published

founding -> funding

Reply:
I would say that the audit was mostly carried by Quarkslab and not the Kuksa team. The Kuksa mostly helped during the scope setting by responding to questions.